### ISSUE TITLE
...

### ISSUE THREAD

**Issue by xFFFFF** [sthewissen/Mynt#69]
https://github.com/QuantConnect/Lean/tree/master/Indicators

**Comment by twixwix** [comment]
Looks very interesting

**Comment by twixwix** [comment]
That looks awesome? Are you using it?

**Comment by LORDofDOOM** [comment]
Both are Apache License, should not be a problem

**Comment by xFFFFF** [comment]
I did not use. I do not want to install all these bots to check them out. I was looking out of curiosity if there is any bot in Perl and I happened to these repos. Structure looking kind like Mynt so I think about add link.

I suggest you search all forks for projects that interest you. It is my rep with the strategies to Gekko, I found 95% of them on on Github: https://github.com/xFFFFF/Gekko-Strategies/tree/master/!WORKSOP
In the repositories I have a script in Bash, which does almost everything, scrape, check sums etc.
